### PR TITLE
Use Rasa REST channel

### LINKED ISSUE #82: loop request to RASA
There are config of parameters:
const POLLING_INTERVAL = 1000;
const WAITING_TIMEOUT = 5000;

I just tested with:
```
      waitingTimeout: "5000",
      pollingInterval: "5000",
```

But it always make request to Rasa backend  every 5 seconds.
How can I make request to Rasa only when user submit the message? or fix this bug?

>  [2019-01-14 15:23:58] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454238133 HTTP/1.1" 200 10516 0.002130
>  [2019-01-14 15:24:04] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454244133 HTTP/1.1" 200 10516 0.002088
>  [2019-01-14 15:24:10] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454250133 HTTP/1.1" 200 10516 0.002085
>  [2019-01-14 15:24:16] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454256134 HTTP/1.1" 200 10516 0.002087
>  [2019-01-14 15:24:22] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454262133 HTTP/1.1" 200 10516 0.002089
>  [2019-01-14 15:24:28] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454268134 HTTP/1.1" 200 10516 0.002091
>  [2019-01-14 15:24:34] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454274134 HTTP/1.1" 200 10516 0.002093
>  [2019-01-14 15:24:40] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454280134 HTTP/1.1" 200 10516 0.002087
>  [2019-01-14 15:24:46] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454286134 HTTP/1.1" 200 10516 0.002084
>  [2019-01-14 15:24:52] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454292134 HTTP/1.1" 200 10516 0.002138
>  [2019-01-14 15:24:58] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454298134 HTTP/1.1" 200 10516 0.002082
>  [2019-01-14 15:25:04] "GET /webhooks/chatroom/conversations/401afeab-0287-432a-a1f8-23f6d6f649c6/log?nocache=1547454304133 HTTP/1.1" 200 10516 0.004063

### LINKED ISSUE #86: Too much time to load rasa_utils.bot
Hi,

When I ran the rasa_utils.bot command as per the help page it started quickly. However, event hough when my hardware configuration has remained the same, the same command now runs very slowly. And also I am unable to see the bot on localhost:5005. It was working before but now it is broken.

I tried running the HTTP API and it is working fine, so I don't think there is a problem with the model. The bot.py keeps hanging up somewhere.

### PR DESCRIPTION
**Pull request by hotzenklotz**
Since many people seem to have trouble connecting the chatroom to their Rasa bot I thought it useful to just use the Rasa-provided REST channel instead. 

#### Advantages:
- super simple configuration for Rasa Core. No Python needed. Just enable the [REST channel](https://rasa.com/docs/core/connectors/#rest-channels) in `credentials.yml` 
- no more polling

#### Disadvantages:
- less customizability of the custom channel. Well, we never did that much
- our custom channel had a simple message store to persist conversations across sessions. The Rasa TrackerStore should basically do the same

#### ToDos:
- [x] remove Python stuff
- [x] update readme etc
- [x] update Debug thingy (?)

@jstriebel @normanrz Before I continue / finish this, any thoughts?

fixes #82
fixes #86

### CODE REVIEW

**Comment by hotzenklotz** [comment]
Good point. I didn't think of fetching the conversation history. Then again, I don't feel this is a terrible loss. We hardly every used that feature. I'll talk to Tom and see if he has an idea of a low-effort workaround.

**Comment by hotzenklotz** [comment]
I finished all changes, cleanup the python stuff, and updated the Readme. As far as I am concerned this PR is ready for merging.

File: src/ConnectedChatroom.js [hunk 1]
```diff
@@ -12,192 +12,172 @@ type ConnectedChatroomProps = {
...
-    );
-    const messages = await res.json();
+  async parseMessages(RasaMessages: Array<RasaMessage>) {
+    const validMessageTypes = ["text", "image", "buttons", "attachment"];
 
-    // Fix dates
-    messages.forEach(m => {
-      m.time = Date.parse(`${m.time}Z`);
-    });
+    debugger;
```
Comment (jstriebel) [inline src/ConnectedChatroom.js]: :bug: :question:

Comment (hotzenklotz) [reply src/ConnectedChatroom.js]: Good catch :-(

**Pull request merged**